crates/ruff_linter/src/rules/airflow/rules/moved_to_provider_in_3.rs

```diff
@@ -112,1271 +112,821 @@ enum Replacement {
 }
 
 fn check_names_moved_to_provider(checker: &mut Checker, expr: &Expr, ranged: impl Ranged) {
```

Comment:
I think we should just pass in the `TextRange` here to avoid the monomorphization of this large function.

Reply:
Just updated it!